src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -414,49 +508,56 @@ def send_comm_message(self, msg_type: str, content: dict) -> None:
         Sends a ipykernel.Comm message to the KBaseJobs channel with the given msg_type
         and content. These just get encoded into the message itself.
         """
+        if msg_type == MESSAGE_TYPE["STATUS"]:
```

Comment:
In `jm.send_comm_message`, `STATUS` responses get `last_checked` added

Reply:
This breaks the model of all backend responses being dictionaries of job data indexed by `job_id`, which is what the FE expects, and it's not what the Jira ticket asked for -- each job should have a `last_checked` field.

I'll see if this data structure can be made to work with the frontend or not.

If you're being completely pedantic, adding the `last_checked` timestamp here also introduces inaccuracy between the `last_updated` job timestamps and this timestamp. In an ideal world, you would take the timestamp from ee2's response to the `check_jobs` request, but the raw response data is hidden away by the JSON RPC interface. Next best would be to retrieve updates from ee2, set a variable with the current timestamp, and then go through your list of jobs using that timestamp as the `last_updated` field.